Tasks/DeployVisualStudioTestAgent/task.json

```diff
@@ -52,7 +52,7 @@
             "defaultValue": "",
             "required": false,
             "helpMarkDown": "Comma separated list of machines where test agent needs to be deployed. For example: 'Machine1,Machine2,Machine3'",
-            "groupName": "agentConfiguration"
+            "groupName": "environment"
```

Comment:
thought environment was taken out of DTL.. shouldnt this be machine group.. please confirm with Ambrish